api/layer/multipart_upload.go

```diff
@@ -562,6 +564,9 @@ func (n *layer) ListParts(ctx context.Context, p *ListPartsParams) (*ListPartsIn
 	parts := make([]*Part, 0, len(partsInfo))
 
 	for _, partInfo := range partsInfo {
+		n.log.Debug("part details",
+			zap.String("cid", p.Info.Bkt.CID.String()),
+			zap.String("oid", partInfo.OID.String()))
```

Comment:
The function can't return any err after line 559, so, `cid` and `oid` will be printed only if `ListParts` passes successfully.

Reply:
We can log `cid` very early for most handlers. I just don't know if we should the logging.
@alexvanin @KirillovDenis what do you think?

Reply:
It seems this is ok. If we couldn't get any parts we cannot print OID.

Reply:
The remaining code (code that goes after two possible errors) should not break at the moment - all the work gets done with the data we already have, and it does not involve any operations with side effects.